packages/freezed/lib/src/templates/concrete_template.dart

```diff
@@ -94,6 +94,31 @@ ${copyWith.abstractCopyWithGetter}
 ''';
   }
 
+  String get _implementations {
+    final implementationTypes = constructor.decorators
+        .where((element) => element.startsWith('@Implements'))
```

Comment:
The parsing should be done outside templates, on Freezed_generator.dart
Potentially with error handling too.